public/index.html

```diff
@@ -25,6 +25,21 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
+    <div id="amazon-root"></div>
+    <script type="text/javascript">
```

Comment:
A comment describing what it is. May be a link to Amazon's developer page will suffice.